content/en/cd-as-a-service/concepts/deployment/how-hpa-works.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "How CD-as-a-Service implements Kubernetes HPA"
+linktitle: "HPA"
+description: >
+  Armory CD-as-a-Service supports Kubernetes deployments with HPA.
+---
+ ## What is HPA?
+ Kubernetes Horizontal Pod Autoscaling (HPA) scales deployments in reaction to changes in observed metrics. When the deployment load increases, the number of deployed pods is increased to meet the demand. The scale behavior is horizontal, rather than vertical, which means that rather than assigning an existing pod more memory or CPU to handle the incoming traffic, new pods that identical instances of your application, are created.
+
+ An HPA resource has three components:
+- A set of one or more metrics, each with a desired target value.
+- A reference to a Kubernetes resource, typically a Deployment but any resource that supports the Scale API sub-resource (e.g., `ReplicaSet`, `StatefulSet`). The HPA scales this resource in response to changes in observed metrics.
+- A minimum and maximum number of replicas.
+
+HPA checks metrics as configured in the deployment file AT A DEFAULT 30 SEC intervals. When the threshold load is met the HPA increases or decreases the number of pods. 
+ 
+ Kubernetes implements Horizontal Auto Scaling (HPA) as a control loop that runs intermittently. `HorizontalPodAutoscaler` controls the scale of a Deployment and its `ReplicaSet`. 
+ 
+ HPA is not recommended for replication controllers. 
+ 
+ HPA does not manage rolling updates by manipulating the replication controller. The deployment object manages the size of underlying replica sets. 
+
+ The sync interval is set by the `kube-controller-manager`. The default interval is 30 seconds. Change the default interval using the `horizontal-pod-autoscaler-sync-period` flag.
+
+The default delay is 3 minutes after an upscale event to allow metrics to stabilize. Change the default upscale delay period using the `horizontal-pod-autoscaler-upscale-delay` flag.
```

Comment:
I'm not sure this "What is HPA" section is even needed since it basically restates (or almost copies verbatim) what's in the Kuberntes docs. What about replacing section with a  prereq section that points to the relevant K8s docs?

Reply:
Please see re-write